OneCommunityGlobal/HighestGoodNetworkApp
# Nathan permission descriptions

# Description
Adds a description to the `putUserProfileImportantInfo` permission, adds timelog management permisison labels to conform to spreadsheet (although not functional), and re-removes `associatedPermissions.js`.

## Main changes explained:
- Changes permission labels and descriptions in `PermissionsConst.js`
- Re-removes `associatedPermissions.js`

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log in as an owner user
5. go to dashboard→ Other Links→ Permissions Management→Any role
6. click on i icon for "Modify Important User Info"
7. verify description is no longer WIP

## Screenshots or videos of changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/23086054/3e2bd00e-2f43-48e3-942a-0ca3e9f53efd)
